README.md

```diff
@@ -8,6 +8,9 @@ For complete documentation and instructions on how to authenticate visit the [de
 
 ### Official
 
+#### Android
+The official Android client library is [yelp-android](https://github.com/yelp/yelp-android) and will help integrate Yelp with Android phones.
```

Comment:
I'm fine with no mention of desktop Java here.  We can add that aspect if people start asking.

:shipit: